fortanix-vme/vme-pkix/src/oid.rs

```diff
@@ -4,10 +4,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#![allow(non_upper_case_globals)]
-
 use pkix::types::ObjectIdentifier;
 
-lazy_static!{
+lazy_static! {
+    /// This OID identifies an extension containing a Nitro attestation report.
     pub static ref ATTESTATION_NITRO: ObjectIdentifier = vec![1, 3, 6, 1, 4, 1, 49690, 2, 2, 6].into();
-}
\ No newline at end of file
+    /// This OID identifies an extension containing a the PCR0 of a Nitro enclave;
+    /// used to insecurely provide identification to debug attestations that
+    /// otherwise contain all zeroes in their PCR0 measurement.
+    pub static ref ATTESTATION_NITRO_PCR0: ObjectIdentifier = vec![1, 3, 6, 1, 4, 1, 49690, 2, 2, 6, 1].into();
```

Comment:
Should we maybe add a reference to the documentation here, explaining where these numbers come from?

Reply:
OIDs are registered; this is a prefix owned by Fortanix; a slight modification of the one we’ve been using for Nitro attestations for a while. This should be added to published documentation, but there’s no formula that derived these; they’re just a sort of registration ID.

Reply:
Can we add this very info to the documentation for `ATTESTATION_NITRO_PCR0`? Just state the fact that it's Fortanix's registration OID. I think this might help developers reading the code